src/update_engine/update_engine_client.cc

```diff
@@ -186,10 +186,14 @@ bool CheckForUpdates() {
 
 static gboolean CompleteUpdateSource(gpointer data) {
   string current_op;
-  if (!GetStatus(&current_op) || current_op == "UPDATE_STATUS_IDLE") {
+  if (!GetStatus(&current_op)) {
     LOG(ERROR) << "Update failed.";
     exit(1);
   }
+  if (current_op == "UPDATE_STATUS_IDLE") {
+    LOG(INFO) << "Update cancelled -- remote version is not newer than the local one";
```

Comment:
```suggestion
    LOG(INFO) << "No update available";
```

Whether the version is newer or not is not really guaranteed. I would also not call it cancelled, but maybe _not available_.

Reply:
make senses - also it's a clearer message :)